app/views/partials/view-claim/add-deduction.html

```diff
@@ -1,22 +1,104 @@
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+<br>
+
 {% if unlock %}
-<h1 class="table-heading">Deduction</h1>
-<table>
-  <tbody>
-    <tr class="form-group{% if errors['deductionType'][0] or errors['deductionAmount'][0] %} form-group-error warning {% endif %}">
-      <td>
-        Type: <select id="deductionType" name="deductionType" class="form-control{% if errors['deductionType'][0] %} form-control-error {% endif %}">
-          <option selected>Select</option>
-          <option value="hc3">{{ displayHelper.getDeductionTypeDisplayName('hc3') }}</option>
-          <option value="overpayment">{{ displayHelper.getDeductionTypeDisplayName('overpayment') }}</option>
-        </select>
-      </td>
-      <td>
-        Amount: <input id="deductionAmount" name="deductionAmount" type="text" class="form-control{% if errors['deductionAmount'][0] %} form-control-error {% endif %}">
-      </td>
-      <td>
-        <input type="submit" formaction="/claim/{{ Claim['ClaimId'] }}/add-deduction" class="button pull-right grey" name="add-deduction" id="add-deduction" value="Add">
-      </td>
-    </tr>
-  </tbody>
-</table>
-{% endif %}
\ No newline at end of file
+    {% set deductionSelect %}
+    <div class="claim-table-deduction-section-container">
+      <div>
+        <span class="govuk-!-margin-top-2 govuk-!-margin-right-4 govuk-!-font-weight-bold">Type</span>
```

Comment:
Could just put the classes on the `div` and remove the `span`? Also if it's bold maybe it's a `heading`?